tests/version.js

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai'
+import validate from '../validate.js'
+import minimalVexDoc from './shared/minimalVexDoc.js'
+
+describe('Version specific tests', function () {
+  it('valid document with basic test', async function () {
+    const tests = Object.values(await import('../versions/basic.js'))
+
+    const result = await validate(tests, {
+      ...minimalVexDoc,
+      vulnerabilities: [
+        ...minimalVexDoc.vulnerabilities,
+        ...minimalVexDoc.vulnerabilities,
+      ],
+    })
+
+    expect(result.isValid).to.be.false
```

Comment:
why would a "valid document" not be valid?
Maybe the test title needs to be changed?